frontend/src/components/MainToolbar.vue

```diff
@@ -211,6 +245,8 @@ export default {
           this.setError({
             message: `Failed to fetch version information. ${err.message}`
           })
+          this.dashboardVersion = this.dashboardVersion || 'Unknown'
```

Comment:
Why do we need this line?

Reply:
If we are not able to fetch the version information I had an endless spinner. I think it is better to show unknown in this case. I don't know I think this worked before (at least the spinner was hidden after timeout)

Reply:
I understand this for the gardener version but not for the dashboard version

Reply:
the dashboard version is also retrieved from the backend

Reply:
Fix version endpoint in backend. Should always return dashboard version.

Reply:
fixed, always return dashboard version